testdata/cheats/Json.t.sol

```diff
@@ -71,9 +71,9 @@ contract ParseJson is DSTest {
     }
 
     function test_nestedObject() public {
-        bytes memory data = cheats.parseJson(json, ".nestedObject");
+        bytes memory data = cheats.parseJson(json, "nestedObject");
```

Comment:
Why does `"nestedObject"` now work? I thought this would have to be `".nestedObject"` per JSONpath/jq syntax. Do both work now?